tests/unit/components/playback/controls/AddCustomer.spec.js

```diff
@@ -372,15 +374,12 @@ describe("AddCustomer.clicked_button", () => {
       invalid_apikey,
       invalid_uuid,
       invalid_nickname,
-      save_btn_css,
-      label
+      save_btn_css
```

Comment:
I don't see the `save_btn_css` being used anywhere in the test. Is this necessary to have?

Reply:
Please search for the following code `expect(save_btn.attributes().style).toBe(save_btn_css);`.  So the reason for validation is the trigger of `'click'` on the non-greyed button its allowed and down the code after successful validation we once again validate the `"save-id"` event. Now that we have removed the spied function `clicked_button` and the value of `choice`. The above verification is required to affirm that the `"save-id"` is emitted from the component. 
Please let me know as it appears necessary to retain save_btn_css.

Reply:
If you click on it and it's greyed out, then it still won't emit the event, right (that's the behavior of the sub-widget)?  So there's not an extra need here to assert things about the CSS is there?  Line 437 would fail if the save button was disabled, because no event would be emitted.

In general, the less we assert about the CSS the better, because that's a styling choice that could change (Ari could decide she wants a slightly different color) which would break our tests even though the logic didn't change.  There will be times when we need to assert things about the CSS because there's no other good way to test the overall logic. But in general we should question each time whether we can accomplish the same goal with asserting less about the CSS or not.